### DISCUSSION TITLE
ERROR on try to create wallet client - 'kzg' is declared here

### DISCUSSION THREAD

**Discussion by unauthxrized** [wevm/viem#2342] [Question]
Wen i write this my IDE show me a errors:
<img width="466" alt="image" src="https://github.com/wevm/viem/assets/42785025/32860d11-e671-45ce-8f3f-636ef6890f2b">
Then i try to run my code:
<img width="484" alt="image" src="https://github.com/wevm/viem/assets/42785025/1a5dd284-857e-422a-a163-e5ccb5eb4e7f">
my imports:
<img width="567" alt="image" src="https://github.com/wevm/viem/assets/42785025/01c4d5a1-9923-43b0-ae42-f2f67647f5d6">

I dont know why this error stalkering me and idk what to do with this

**Comment by tmm** [comment]
What TypeScript version are you using?

**Comment by judeProground** [comment]
@unauthxrized Hi, I'm experiencing the same problem. Have you found the cause?
Here are the versions.
```
    "viem": "^2.21.0",
    "@types/node": "^20.14.10",
    "typescript": "^5.5.3"
```

**Reply by jxom** [reply]
please reproduce it here and send back the link:

https://www.typescriptlang.org/play/?#code/JYWwDg9gTgLgBAbzgYygUwIYzQdQwG3zRgGF9g0A7GAGjgAsYYw4BfOAMyghDgHIAbhRB8AUKEixEcMFGACsaANJoAngBUIAQWTIIAV2ptO3XoOEB6DLoPUAzmInR4SEBmCVKxY1x78haCAWyPTulA6ionrh8MjkVPAAvCjoiniExGQU1AAUCKJwhXDWeoYwAFwycgrYKhraNmU5fAAMAB58AJQ0BUUhYZVuHl60vYUwUBjhkhUMTGA5naKsS1EQMQwYdvRwyXHZMAB0dlQAJuqT4dYwwOt5Y3AwEJWtbS3vH59f3z+-n2IrIA

**Reply by AndonMitev** [reply]
@jxom i do have similar problem

stack:

```
"viem": "^2.21.32",
"typescript": "5.6.3"
```

<img width="517" alt="image" src="https://github.com/user-attachments/assets/be611734-4655-4314-9554-3403ef45935d">


thats all the prop i see, any ideas what can cause the issue?

**Comment by colfax23** [comment]
I ran into this too I was able to fix it by making sure to set `"strict": true` in my tsconfig.  It looks like when strict mode is not enabled, typescript is struggling to decipher the correct type here, thus resulting in an incorrect error.  Hope this helps anyone else who runs into this!

**Reply by QingyangKong** [reply]
Thank you. This solved my problem, but I am still wondering why
